preprocessing/__init__.py

```diff
@@ -2,36 +2,62 @@
 from color_refiment_helper import compare
 
 
-def check_graph_order(g: Graph, f: Graph):
+def checks(g, h) -> bool:
+    """
+    Collection method of all preprocessing checks
+
+    :param g: Graph
+    :param h: Graph
+    :return: Boolean: True if everything checks out
+    """
+    return check_degrees(g, h) and check_graph_size(g, h) and check_graph_order(g, h)
+
+
+def check_graph_order(g: Graph, h: Graph):
     """
     This method checks the orders (amount of vertices) of the graphs are the same
 
     :param g: Graph
-    :param f: Graph
+    :param h: Graph
     :return: Boolean: True if the amount of vertices are the same
     """
-    return len(g.vertices) == len(f.vertices)
+    return len(g.vertices) == len(h.vertices)
 
 
-def check_graph_size(g: Graph, f: Graph):
+def check_graph_size(g: Graph, h: Graph):
     """
     This method checks if the sizes (the amount of edges) of the graphs are the same
 
     :param g: Graph
-    :param f: Graph
+    :param h: Graph
     :return: Boolean: True if the amount of edges are the same
     """
-    return len(g.edges) == len(f.edges)
+    return len(g.edges) == len(h.edges)
 
 
-def check_degrees(g: Graph, f: Graph):
+def check_degrees(g: Graph, h: Graph):
     """
     This method checks if the degrees of all the vertices in the graphs are all the same
 
     :param g: Graph
-    :param f: Graph
+    :param h: Graph
     :return: Boolean: True if the degrees are the same
     """
     degree_g = [v.degree for v in g.vertices]
-    degree_f = [v.degree for v in f.vertices]
-    return compare(degree_g, degree_f)
+    degree_h = [v.degree for v in h.vertices]
+    return compare(degree_g, degree_h)
+
+
+def remove_loners(g: Graph):
```

Comment:
Wat als g+h 1 loner heeft de uit g komt? Wordt deze dan hier weggehaald en krijgen we verkeerde antwoorden?

Reply:
als g + h 1 loner heeft, dan zit ie dus in óf g, óf h. Dus misschien eerst de count-vertices-check. Dan zijn de graphs sowieso niet isomorph, maar ik zal er nog een test om heen schrijven om dat te bewijzen.